src/WingetCreateCLI/Properties/Resources.resx

```diff
@@ -611,4 +609,24 @@
   <data name="ModifyOptionalFields_Message" xml:space="preserve">
     <value>Would you like to modify the optional fields?</value>
   </data>
+  <data name="InstallerDetectedFromUrl_Message" xml:space="preserve">
+    <value>{0} {1} installer detected from the url: {2}</value>
+    <comment>{0} - will be replaced with installer architecture
+{1} - will be replaced with the installer type 
+{2} - will be replaced with the installer url</comment>
+  </data>
+  <data name="MultipleMatchingInstallerNodes_Error" xml:space="preserve">
+    <value>Multiple existing installer nodes matched the following installer(s):</value>
+  </data>
+  <data name="NewInstallerUrlMustMatchExisting_Message" xml:space="preserve">
+    <value>Each new installer URL must have a match to an existing installer node based on installer type and architecture.</value>
+  </data>
+  <data name="DetectedArchMismatch_Message" xml:space="preserve">
+    <value>The detected architecture might be different than what is specified in the installer URL for the following installers:</value>
+  </data>
+  <data name="InstallerBinaryMismatch_Message" xml:space="preserve">
+    <value>Selected architecture detected from URL: {0} | Architecture detected from binary: {1}</value>

```

Comment:
I think I'd change this to something like:

Using architecture detected from URL ({0}), overriding architecture detected from binary ({1})

What do you think?

Reply:
I like that better, more clear about the fact that we overrode the architecture. Change to suggested.